types/google.visualization/index.d.ts

```diff
@@ -1166,6 +1166,48 @@ declare namespace google {
             width?: number;
         }
 
+        //#endregion
+        //#region Dashboard
+
+        // https://developers.google.com/chart/interactive/docs/gallery/controls#dashboard
+        export class Dashboard {
+            constructor(containerRef: HTMLElement);
+            bind(controls: any, charts: ChartWrapper): google.visualization.Dashboard;
```

Comment:
As per the [doc](https://developers.google.com/chart/interactive/docs/gallery/controls#dashboard)

> controls - Either a single one or an array of google.visualization.ControlWrapper instances defining the controls to bind

> charts - Either a single one or an array of google.visualization.ChartWrapper instances defining the charts the that will be driven the by the controls

both controls and charts are either single or array of type google.visualization.ControlWrapper